Cargo.toml

```diff
@@ -5,15 +5,15 @@ authors = [
     "Kaiden42 <gitlab@tinysn.com>",
     "Andrew Wheeler <genusistimelord@gmail.com>",
 ]
-edition = "2021"
+edition = "2024"
```

Comment:
the MSRV and edition shift probably belong in separate MRs or at least separate commits